norona_marynicole/index.html

```diff
@@ -0,0 +1,187 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link href="style.css" rel="stylesheet" />
+    <title>Norona, Mary Nicole</title>
+  </head>
+  <body>
+    <div id="my_name">
+      <h1>NORONA, MARY NICOLE</h1>
+    </div>
+
+    <div>
+      <h2 class="countries">COUNTRIES</h2>
+    <a href="./countries.html" id="countries-pages">Countries</a>
+    </div>
+
+    <div>
+      <h2 class="title">MY INTEREST AND HOBBIES</h2>
+      <ul>
+        <li>Watching Movies</li>
+        <li>Reading Books</li>
+        <li>Playing Mobile Games</li>
+        <li>Playing with my nephew</li>
+        <li>Dancing Tiktok</li>
+      </ul>
+    </div>
+
+    <div>
+      <h2 class="title">MY WORKS</h2>
+      <ol>
+        <li><a href="1.html">html</a></li>
+        <li><a href="2.html">html</a></li>
+        <li><a href="3.html">html</a></li>
+        <li><a href="4.html">html</a></li>
+        <li><a href="5.html">html</a></li>
+      </ol>
+    </div>
+
+    <div>
+      <h2 class="goals">MY GOALS</h2>
+      <ol>
+        <li>To finish my studies</li>
+        <li>To have a stable and great job</li>
+        <li>To be financially stable</li>
+        <li>To give a comfortable life to my family</li>
+        <li>To be rich and successful in life</li>
+      </ol>
+
+      <h2 class="title">COMMENTS OF MY TEAMMATES</h2>
+      <ul>
+        <li>
+          <p>
+            Your goals for education, stability, financial security, and family
+            well-being reflect ambition and determination, promising a future of
+            success and fulfillment. - Bergado, Shamma
+          </p>
+        </li>
+        <li>
+          <p>
+            Your goals perfectly represent your values and desires. I'm thrilled
```

Comment:
There should be max of 80 col per line.

Follow the guidelines
https://docs.google.com/presentation/d/1iCoz17mBxZtIQ9PifAW0jjKsuv4xDTlx-TPQxdLySow/edit#slide=id.g24da8de92d7_0_0